docs/index.md

```diff
@@ -115,7 +115,7 @@ provider "project" {
 <!-- schema generated by tfplugindocs -->
 ## Schema
 
-### Optional
+### Required
```

Comment:
There's a [bug](https://github.com/hashicorp/terraform-plugin-docs/issues/42) in the tfplugindocs that marks attributes as 'Optional' if there's 'DefaultFunc' set.